src/components/ErrorPopup/ErrorPopup.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const ErrorPopup = ({ message }) => {
+  return (
+    <div
+      className={`error-message${message ? ' error-message--active' : null}`}
```

Comment:
What do you think about something like `const classes = ['error-message', message && 'error-message--active'].filter(Boolean)` and then `className={classes.join(' ')}`? Same for other places below, where classNames are constructed on-the-fly? It might be more readable, if we construct them earlier, imho 😉

Reply:
Sure, I will try it out